src/components/layouts/EditProfileForm/index.tsx

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import EditProfileForm from '../EditForm';
+import Input from '../../ui/Input';
+
+const EditProfileFormWrapper = props => {
+  const [firstName, setFirstName] = React.useState();
+  const [lastName, setLastName] = React.useState('');
+  const [email, setEmail] = React.useState('');
+  const [userName, setUserName] = React.useState('');
+
+  const handleSubmit = (): void => {
+    const userData = {
+      firstName,
+      lastName,
+      email,
+      userName,
+    };
+  };
+
+  return (
```

Comment:
I think this belongs to another card. If so, it shouldn't be on this PR.

Reply:
Though the handle submit function isn't necessary for my task but, I  think it makes it easier for whoever is going to implement the functionality.

Reply:
resolved